extension/apmconfigextension/README.md

```diff
@@ -64,24 +64,80 @@ unique remote configuration.
 
 ![Extension workflow](./extension-workflow.png "Extension workflow")
 
-## Extension sample configuration
+## Getting started
+
+All that is required to enable the apmconfig extension is to include it in the extensions definitions:
 
 ```
 extensions:
-  basicauth:
-   client_auth:
-     username: changeme
-     password: changeme
-
-apmconfig:
-  agent_config:
-   elasticsearch:
-     endpoint: "https://127.0.0.1:9200"
-     tls:
-       ca_file: path_to_ca-cert.pem
+  bearertokenauth:
+    scheme: "APIKey"
+    token: "<ENCODED_ELASTICSEACH_APIKEY>"
+
+  apmconfig:
+    agent_config:
+     elasticsearch:
+       endpoint: "<ELASTICSEACH_ENDPOINT>"
        auth:
-         authenticator: basicauth
-  opamp:
-    server:
-      endpoint: ":4320"
+         authenticator: bearertokenauth
+    opamp:
+      protocols:
+        http:
+          endpoint: ":4320"
+```
+
+
+## Advanced configuration
+
+The apmconfig extension embeds the [confighttp.ServerConfig](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/README.md), which means it supports standard HTTP server configuration, including TLS/mTLS and authentication.
```

Comment:
Can we pin those to specific versions instead of `main`?

Reply:
Definitely, https://github.com/elastic/opentelemetry-collector-components/pull/596/commits/2e623d3ff9697a35dddef3b43ca3a322d5f8d28b